src/k8s-extension/azext_k8s_extension/consts.py

```diff
@@ -13,11 +13,15 @@
 CONNECTED_CLUSTER_RP = "Microsoft.Kubernetes"
 MANAGED_CLUSTER_RP = "Microsoft.ContainerService"
 APPLIANCE_RP = "Microsoft.ResourceConnector"
+PROVISIONED_CLUSTER_RP = "Microsoft.HybridContainerService"

```

Comment:
This is being added separately by @NarayanThiru in a separate PR for `provisionedClusters`, do you require `provisionedClusters` RP to be part of the this PR?